docs/src/main/paradox/docs/fusion/project.md

```diff
@@ -16,45 +15,38 @@ The Nexus Fusion admin app allows you to manage your Nexus Delta instance. In pa
 
 When creating a project, you can use "Advanced Settings" to set Base, Vocab and API mapping for the project.
 
-Base is an IRI used as a @link:[curie](https://www.w3.org/TR/2010/NOTE-curie-20101216/){ open=new } for generating
-'@id' of this child resources of the project.
+Base is an IRI used as a @link:[curie](https://www.w3.org/TR/2010/NOTE-curie-20101216/){ open=new } for generating '@id' of this child resources of the project.
 
-Vocab is an IRI used as a @link:[curie](https://www.w3.org/TR/2010/NOTE-curie-20101216/){ open=new } prefix for all
-unqualified predicates in child resources of the project. JSON object - provides a convenient way to deal with URIs
-when performing operations on a sub-resource. This field is optional.
+Vocab is an IRI used as a @link:[curie](https://www.w3.org/TR/2010/NOTE-curie-20101216/){ open=new } prefix for all unqualified predicates in child resources of the project. JSON object - provides a convenient way to deal with URIs when performing operations on a sub-resource. This field is optional.
 
 More info can be found @ref:[here](../delta/api/projects-api.md#api-mappings) regarding API Mappings.
 
-### Creating a New Organization
+### Creating a New Project
+
+The current version allow the user to create project in different main parts of the application, organization page, projects page, and also in the global search types pages.
 
 @@@ div { .center }
-![](../assets/try-nexus-sandbox-admin-create-org-form.png)
-@@@
 
-### Creating a New Project
+![Project creation modal](assets/fusion-projects-create-prj.png)
 
-@@@ div { .center }
-![](../assets/try-nexus-sandbox-admin-organization.png)
 @@@
 
-@@@ div { .center }
-![](../assets/try-nexus-sandbox-admin-create-project-form.png)
-@@@
+The project creation modal is configurable in way that can allow the user to select an organization when the user is not on the one of his organizations page.
```

Comment:
* is not *on one* of *their* organizations page. 

Additionally, we may also want to explicitly mention if the user has to be an admin of this organization, or they can be just a member.